### ISSUE TITLE
[DevEx]: Refactor tests to make more consistent use of userFixture data

### ISSUE THREAD

**Issue by itsalaidbacklife** [cuttle-cards/cuttle#344]
### Improvement Summary

The e2e tests make somewhat inconsistent use of the hard coded usernames and passwords used to log in. We should centralize all these references into the existing `tests/e2e/fixtures/userFixtures.js` file and update the commands to simplify their usage.

### Detailed Description

The `tests/e2e/fixtures/userFixtures.js` file exports several objects that each have a `username` and `password` object:

```
// tests/e2e/fixtures/userFixtures.js
const playerOne = {
  username: 'Player1',
  password: 'player1DaBest',
};
```

Yet we also redundantly define a `username`, `validPassword`, `opponentUsername`, and `opponentPassword` for export in `tests/e2e/support/helpers.js`. We should move these into the same file and format as the rest of the userFixtures. 

Then we should clean up the commands that _use_ these fixtures i.e. `signupOpponent()`, `signupPlayer()`, `loginPlayer()` so that instead of taking two arguments (`username`, and `password`), they input a single object parameter that itself has `username` and `password` attributes so that they can be called directly with the fixtures imported from the userfixture.

That way consumption of the user fixtures would look like this:

```
import { playerOne } from '../../fixtures/userFixtures';

cy.signupOpponent(playerOne);
```

instead of the current setup:

```
import { playerOne } from '../../fixtures/userFixtures';

cy.signupOpponent(playerOne.username, playerOne.password);
```

**Comment by DonPierreDon** [comment]
I'd like to take this issue

**Comment by itsalaidbacklife** [comment]
Resolved by #388
